fetch-utils.js

```diff
@@ -0,0 +1,25 @@
+const SUPABASE_URL = 'https://usheruhymluidrfdhheu.supabase.co';
+const SUPABASE_KEY =
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InVzaGVydWh5bWx1aWRyZmRoaGV1Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2NzQyNTc1MDksImV4cCI6MTk4OTgzMzUwOX0.QjjqpyfcTdEDXQrrL5etMEt5BZFjp0GeTudUiqwVKV8';
+
+const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
+
+export async function getGames() {
+    const { data, error } = await client.from('games').select(`*, game_genre (genre)`);
```

Comment:
I think I was wrong about this line during your TA request. If you are only grabbing info on that main page from the games table, you should not need the `game_genre (genre)` part...sorry!